sections/07_conclusions.md

```diff
@@ -64,11 +82,12 @@ transcription factor binding prediction and molecular graphs instead of
 pre-computed bit vectors for drug discovery. These flexible input feature
 representations have spurred creative modeling approaches that would be
 infeasible with other machine learning techniques. Unsupervised methods are
-currently less-developed than their supervised counterparts, but they may have
-the most potential. When deep learning algorithms can summarize very large
-collections of input data into interpretable models that spur scientists to ask
-questions that they didn't know to ask, it will be clear that deep learning has
-transformed biology and medicine.
+currently less developed than their supervised counterparts, but they may have
+the most potential because of how expensive and time-consuming it is to label
```

Comment:
I know we've had / are having this discussion elsewhere, but unsupervised methods are valuable for many reasons, primary of which is that they are revealing the truth / otherwise unseen divisions rather than reinforcing a human-dictated classification. 

How to say this compactly is an open question. Perhaps in the later stages we need to look for all mentions of "unsupervised" and put in a single clear statement.